app/views/admin/super_admin/super_admin.py

```diff
@@ -16,10 +16,11 @@ def index_view(self):
         super_admins = DataGetter.get_all_super_admins()
         admins = DataGetter.get_all_admins()
         registered_users = DataGetter.get_all_registered_users()
-        organizers = []
-        co_organizers = []
-        track_organizers = []
-        attendees = []
+        # TODO Fix function and correct this
+        organizers = 0
```

Comment:
Yes. I have now hard coded this to zero temporarily. Will correct it once I rewrite the methods get_all_organizers, get_all_co_organizers, get_all_track_organizers, and get_all_attendees in app/helpers/data_getter.py that are used to get a count of the same.